components/treeview/events.md

```diff
@@ -78,6 +79,16 @@ The event handler receives as an argument an `TreeViewItemRenderEventArgs` objec
 
 The `SelectedItemsChanged` event fires when the [selection]({%slug treeview-selection-overview%}) is enabled and the user clicks on a new item.
 
+## Drag Events
+
+The TreeView implements the Drag and Drop functionality through the following drag events:
```

Comment:
Prefer active voice "fires" instead of "is triggered".

```suggestion
The TreeView implements the Drag and Drop functionality through the following drag events:

```